### PR TITLE
Updating context passed for marketplace validation

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

- Making some updates to the context that we pass in the output of the marketplace validation command.
- I updated the output in the CLI to make it a little easier to read.
- I updated the copy of the commands to make them more actionable and more consistent with each other.

## Screenshots

Updated CLI output
image

## Who to Notify

cc/ @KatzMitch

### CODE REVIEW

**Comment by brandenrodgers** [comment]
I updated the title of each grouped section in the CLI output to be bold. 
image

**Comment by KatzMitch** [comment]
I love this new formatting!

**Comment by PaulJKim** [comment]
This looks amazing! Thanks Branden!

File: packages/cli/lib/validators/logValidatorResults.js [hunk 1]
```diff
@@ -26,27 +29,31 @@ function logResultsAsJson(results) {
...
-        break;
-      case VALIDATION_RESULT.SUCCESS:
-        logger.success(validatorName);
-        break;
-      default:
-        logger.log(message);
-    }
+  groupedResults.forEach(results => {
+    logger.log(chalk.bold(`${results[0].validatorName} validation results:`));
+    logger.group();
```
Comment (miketalley) [inline packages/cli/lib/validators/logValidatorResults.js:38]: 👍

File: packages/cli/lib/validators/marketplaceValidators/theme/ThemeConfigValidator.js [hunk 2]
```diff
@@ -98,6 +100,6 @@ class ThemeValidator extends BaseValidator {
 }
 
 module.exports = new ThemeValidator({
-  name: 'Theme',
-  key: 'theme',
+  name: 'Theme config',
+  key: 'themeConfig',
```
Comment (miketalley) [inline packages/cli/lib/validators/marketplaceValidators/theme/ThemeConfigValidator.js:104]: Would it make sense to have these in an enum?

**Review by miketalley — APPROVED** [review]
Just one question, but definitely not a blocker. Code looks good!

File: packages/cli/lib/validators/marketplaceValidators/theme/ThemeConfigValidator.js (hunk 2 above)
Comment (brandenrodgers) [reply packages/cli/lib/validators/marketplaceValidators/theme/ThemeConfigValidator.js]: Good idea, done in 2359c16

**Pull request merged**